minitests/iostandard/features/Makefile

```diff
@@ -0,0 +1,76 @@
+PART=xc7a50tfgg484-1
+VIVADO_PART=xc7a50tfgg484-1
+
+BIT2FASM_ARGS= --part "$(XRAY_DIR)/database/artix7/$(PART)" --verbose
```

Comment:
`artix7` should probably be `XRAY_DATABASE`?  See https://github.com/SymbiFlow/prjxray/blob/master/settings/artix7.sh#L1

Reply:
Fixed.